packages/headless/doc-parser/src/entity-builder.ts

```diff
@@ -101,12 +101,8 @@ function getSummary(comment: DocComment | undefined) {
     const nodes = removeBlockTagNodes(comment.summarySection.nodes);
     return emitAsTsDoc(nodes);
   } catch (e) {
-    console.log(
-      'failed to get summary for comment:\n',
-      comment.emitAsTsdoc(),
-      e
-    );
-    return '';
+    const message = `failed to get summary for comment:\n\n${comment.emitAsTsdoc()}\n${e}`;
+    throw new Error(message);
```

Comment:
The TSDoc parser was not able to parse the date format in the summaries below. The build did not fail on CI because the error was being caught and logged, rather than rethrown.